sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/ExpressionParserSuite.scala

```diff
@@ -778,11 +778,11 @@ class ExpressionParserSuite extends AnalysisTest {
 
   test("SPARK-17832 function identifier contains backtick") {
     val complexName = FunctionIdentifier("`ba`r", Some("`fo`o"))
-    assertEqual(complexName.quotedString, UnresolvedAttribute("`fo`o.`ba`r"))
+    assertEqual(complexName.quotedString, UnresolvedAttribute("```fo``o`.```ba``r`"))
```

Comment:
nit: `UnresolvedAttribute(Seq("`fo`o", "`ba`r"))` to make it more obvious

Reply:
Thanks. I've done for here and the other place.